cel/cos_tlv.go

```diff
@@ -0,0 +1,34 @@
+package cel
+
+import "crypto"
+
+const (
+	// CosTypeValue indicate the CELR event is a COS content
+	// TODO: the value needs to be reserved in the CEL spec
+	CosTypeValue uint8 = 88
+)
+
+// CosTlv is a specific event type created for the COS (Google Container-Optimized OS),
+// used as a CEL content.
+type CosTlv TLV
```

Comment:
Discussed offline, we will want an implemenation that looks like:

```go
type CosTlv struct {
	data []byte // Actual a nested TLV
}

func (c CosTlv) GetTLV() TLV {
	return TLV{
		Type: CosTypeValue,
		Length: len(c.data),
		Value: c.data,
	}
}

func (c CosTlv) GenerateDigest(hashAlgo crypto.Hash) ([]byte, error) {
	b, err := c.GetTLV().Marshal()
	if err != nil {
		return nil, err
	}

	hash := hashAlgo.New()
	if _, err = hash.Write(b); err != nil {
		return nil, err
	}
	return hash.Sum(nil), nil
}
```

Reply:
done